micromanager_gui/main_window.py

```diff
@@ -213,7 +213,6 @@ def load_cfg(self):
         cfg = self.cfg_wdg.cfg_LineEdit.text()
         if cfg == "":
             cfg = "MMConfig_demo.cfg"
-            self.cfg_wdg.cfg_LineEdit.setText(cfg)
```

Comment:
Why remove this?

Reply:
@ianhi  Sorry, my mistake when fixing conflicts! I'll put it back

Reply:
actually... do we really want to set it?  I see that we need to fallback to the string `"MMConfig_demo.cfg"` if nothing was provided, but why actually set the text in the LineEdit?  I kinda like that it's just placeholder text until a user manually enters something...